python/tests/estimators/test_keras_estimators.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import print_function
+
+import os
+import shutil
+import sys
+import tempfile
+
+import PIL.Image
+import numpy as np
+import tensorflow as tf
+from keras.applications import Xception
+from keras.applications.imagenet_utils import preprocess_input
+
+from pyspark import SparkContext
+from pyspark.sql import DataFrame, Row
+from pyspark.sql.functions import udf
+
+from sparkdl.estimators.keras_image_file_estimator import KerasImageFileEstimator
+
+from ..tests import SparkDLTestCase
+from ..transformers.image_utils import _getSampleJPEGDir, getSampleImagePaths
+
+def _load_image_from_uri(local_uri):
+    img = (PIL.Image
+           .open(local_uri)
+           .convert('RGB')
+           .resize((299, 299), PIL.Image.ANTIALIAS))
+    img_arr = np.array(img).astype(np.float32)
+    img_tnsr = preprocess_input(img_arr[np.newaxis, :])
+    return img_tnsr
+
+class KerasEstimatorsTest(SparkDLTestCase):
+
+    def test_valid_params(self):
+        temp_dir = tempfile.mkdtemp()
+        model_filename = os.path.join(temp_dir, 'model.h5')
+
+        estimator = (KerasImageFileEstimator()
```

Comment:
Btw, we generally pass named arguments to constructors in Python, rather than using setters; either is Ok though